src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -207,7 +207,7 @@ internal sealed class MeterProviderSdk : MeterProvider
                             }
                         }
 
-                        OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent($"Succeeded publishing Instrument = \"{instrument.Name}\" of Meter = \"{instrument.Meter.Name}\".");
+                        OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent($"Completed publishing Instrument = \"{instrument.Name}\" of Meter = \"{instrument.Meter.Name}\".");
```

Comment:
Completed is more apt here. Whether success/failure depends on other things, and are appropriately logged elsewhere.